### PR TITLE
Add debug and release parameters to the ci build scripts

### PR DESCRIPTION
**Pull request by mmitche**
Allows for us to properly build a non debug configuration in CI testing

/cc @jaredpar

### CODE REVIEW

File: cibuild.sh [hunk 1]
```diff
@@ -34,6 +35,14 @@ do
         FULL_RUN=false
         shift 1
         ;;
+		--debug)
```
Comment (jaredpar) [inline cibuild.sh]: Can you switch to spaces?

File: cibuild.cmd [hunk 2]
```diff
@@ -1,17 +1,30 @@
...
-msbuild /nologo /v:m /m src/Compilers/CSharp/csc2/csc2.csproj
-msbuild /nologo /v:m /m src/Compilers/VisualBasic/vbc2/vbc2.csproj
+msbuild /nologo /v:m /m src/Compilers/Core/VBCSCompiler/VBCSCompiler.csproj /p:Configuration=%BuildConfiguration%
+msbuild /nologo /v:m /m src/Compilers/CSharp/csc2/csc2.csproj /p:Configuration=%BuildConfiguration%
+msbuild /nologo /v:m /m src/Compilers/VisualBasic/vbc2/vbc2.csproj /p:Configuration=%BuildConfiguration%
 
 mkdir Binaries\Bootstrap
-move Binaries\Debug\* Binaries\Bootstrap
+move Binaries\%BuildConfiguration%\* Binaries\Bootstrap
 msbuild /v:m /t:Clean src/Toolset.sln
```
Comment (pharring) [inline cibuild.cmd]: Also needs ```/p:Configuration=%BuildConfiguration%```

File: cibuild.sh [hunk 3]
```diff
@@ -114,7 +123,7 @@ clean_roslyn()
 {
     echo Cleaning the enlistment
     xbuild /v:m /t:Clean src/Toolset.sln
```
Comment (pharring) [inline cibuild.sh]: Also needs ```/p:Configuration=$BUILD_CONFIGURATION```

**Comment by mmitche** [comment]
Done.

**Comment by jaredpar** [comment]
LGTM

**Comment by pharring** [comment]
:+1:

**Pull request merged**